io_test.go

```diff
@@ -302,6 +308,47 @@ func TestOutputEquation(t *testing.T) {
 	DeleteShapefile(TestOutputFilename)
 }
 
+func BenchmarkOutput(b *testing.B) {
+	cfg, ctmdata, pop, popIndices, mr := VarGridData()
+
+	emis := NewEmissions()
+	emis.Add(&EmisRecord{
+		PM25: E,
+		Geom: geom.Point{X: -3999, Y: -3999.},
+	}) // ground level emissions
+
+	o, err := NewOutputter(TestOutputFilename, false, map[string]string{
+		"TotalPop":   "TotalPop",
+		"WhiteNoLat": "WhiteNoLat",
+		"NPctWNoLat": "{sum(WhiteNoLat) / sum(TotalPop)}",
+		"NPctOther":  "1 - NPctWNoLat",
+		"TotalPopD":  "coxHazard(loglogRR(TotalPM25), TotalPop, MortalityRate)",
+		"TotalPM25":  "TotalPM25",
+		"PM25Emiss":  "PM25Emissions",
+		"BasePM25":   "BaselineTotalPM25",
+		"WindSpeed":  "WindSpeed"},
+		nil)
+	if err != nil {
+		b.Error(err)
+	}
+
+	d := &InMAP{
+		InitFuncs: []DomainManipulator{
+			cfg.RegularGrid(ctmdata, pop, popIndices, mr, emis),
+			o.CheckOutputVars(),
+		},
+		CleanupFuncs: []DomainManipulator{
+			o.Output(),
+		},
+	}
+	if err := d.Init(); err != nil {
+		b.Error(err)
+	}
+	if err := d.Cleanup(); err != nil {
```

Comment:
Then I think we can make several subbenchmarks using b.Run().  One subbenchmark can surround d.init, one can surround newoutputter, and can surround d.cleanup.